otterdog/eclipse-kura.jsonnet

```diff
@@ -1,10 +1,40 @@
 local orgs = import 'vendor/otterdog-defaults/otterdog-defaults.libsonnet';
 
-local customBranchProtectionRule(name) = 
-  orgs.newBranchProtectionRule(name) {
-    dismisses_stale_reviews: true,
-    require_last_push_approval: true,
-    required_approving_review_count: 1,
+local customDevRuleset(name) =
+  orgs.newRepoRuleset(name) {
+    bypass_actors+: [
+      "@eclipse-kura/iot-kura-project-leads"
+    ],
+    include_refs+: [
+      std.format("refs/heads/%s", name),
+    ],
+    required_pull_request: {
+        dismisses_stale_reviews: true,
+        requires_last_push_approval: true,
+        required_approving_review_count: 1,
+    },
+    required_status_checks+: [
+      "Validate PR title",
```

Comment:
I would not add these checks to the general ruleset template, otherwise they would be required everywhere, which will probably not be what you want.